stable/aws-load-balancer-controller/README.md

```diff
@@ -142,4 +142,5 @@ The following tables lists the configurable parameters of the chart and their de
 | `serviceAccount.name`              | Service account to be used                                | None                                                                                       |
 | `terminationGracePeriodSeconds`    | Time period for controller pod to do a graceful shutdown  | 10                                                                                         |
 | `ingressClass`                     | The ingress class to satisfy                              | alb                                                                                        |
+| `region`                          | Set to use KIAM or kube2iam for example                             |       
```

Comment:
Lets say something like this in the description, -
The AWS region for the kubernetes cluster.